RecoJets/JetProducers/python/PileupJetID_cfi.py

```diff
@@ -34,6 +35,7 @@
 #     residualsTxt     = cms.FileInPath("RecoJets/JetProducers/data/download.url") # must be an existing file
 )
 run2_miniAOD_UL.toModify(pileupJetId, algos = _chsalgos_106X_UL17)
+run2_miniAOD_devel.toModify(pileupJetId, algos = _chsalgos_106X_UL18)
```

Comment:
from https://github.com/cms-sw/cmssw/pull/32872#discussion_r574722240

Ehm, it may be better that run2_jme_2018 actually existed.
Otherwise, the last line would have (run2_miniAOD_devel & (~run2_jme_2016) & (~run2_jme_2017)).

Reply:
okay, somehow I missed this comment. I will push another quick fix.